src/main/java/com/github/redouane59/twitter/helpers/TweetStreamConsumer.java

```diff
@@ -0,0 +1,136 @@
+package com.github.redouane59.twitter.helpers;
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+import java.nio.charset.StandardCharsets;
+import java.util.List;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.github.redouane59.twitter.IAPIEventListener;
+import com.github.redouane59.twitter.TwitterClient;
+import com.github.redouane59.twitter.dto.tweet.TweetV2;
+import com.github.scribejava.core.model.Response;
+
+import lombok.extern.slf4j.Slf4j;
+
+/**
+ * Consumes the data from the buffers received from twitter
+ */
+@Slf4j
+public class TweetStreamConsumer {
+
+  private StringBuilder buffer;
+
+  public TweetStreamConsumer() {
+    this.buffer = new StringBuilder();

```

Comment:
Can you instantiate it directly outside the constructor ? 
see https://stackoverflow.com/a/3919225/2337334

Reply:
That's a habit i have with other languages :). But i get the point.